geometry/geometry_config.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import print_function
 import shipunit as u
 import ROOT as r
+import math
```

Comment:
```suggestion
```

Reply:
Thanks, but the call to math I mentioned should be replaced, otherwise it would not compile.